recycle-node.rb

```diff
@@ -39,6 +39,10 @@ def main
   wait_for_node_to_be_replaced
 end
 
+def config_cluster
+  execute "kubectl config use-context #{K8S_CLUSTER_NAME}"
```

Comment:
Sorry, one more thought. If this fails (e.g. there is no such context defined in the user's .kube/config file), the script will continue executing, and could delete a node in the wrong cluster.

This should have a guard where, unless the command returns successfully, we raise an error and die.